assets/app/scripts/controllers/project.js

```diff
@@ -88,58 +67,59 @@ angular.module('openshiftConsole')
         var servicePods = [];
         for (var selectorKey in service.selector) {
           var selectorValue = service.selector[selectorKey];
-          var pods = $scope.podsByLabel[selectorKey][selectorValue];
-          for (var podId in pods) {
-            var pod = pods[podId];
-            servicePods.push(pod);
+          if ($scope.podsByLabel[selectorKey]) {
+            var pods = $scope.podsByLabel[selectorKey][selectorValue];
+            for (var podId in pods) {
```

Comment:
not sure what the behavior is if `pods` is null... firefox seems to work, but not sure we can rely on that